### ISSUE TITLE
Device list no longer appears on node-red 

### ISSUE THREAD

**Issue by Anthom12** [hardillb/node-red-alexa-home-skill-web#128]
Hello,
I wanted to add a new device to my account and I just saw that on Node-red the list of devices in my account no longer appears. Previously created devices still work but I cannot select a new one. 
Can you help me please? 
![image](https://user-images.githubusercontent.com/95219857/143857131-62cd4802-3fc8-4454-94fe-64f03885db3b.png)

**Comment by hardillb** [comment]
I assume that the list still shows all the devices on the website?

What happens if you hit the refresh button next to the drop down?

**Comment by Anthom12** [comment]
On the website everything seems normal 
![image](https://user-images.githubusercontent.com/95219857/143863835-2cc8e6e5-bab2-479a-827b-e463976154fd.png)
If I press the refresh button it doesn't change anything.
On nodered there is nothing written in debug.
Thank you for your quick reply

**Comment by hardillb** [comment]
No, not the debug sidebar, the actual console logs output from Node-RED.

It should log if it fails to pull down the list of devices

You can also try removing the comment from the start of line 327 of the `alexa.js` file to get a better error message.

**Comment by Anthom12** [comment]
Ok I found the logbut he does not move when I refresh the device list.
If I uncomment line 327, nodered does not start anymore.
![image](https://user-images.githubusercontent.com/95219857/143897054-805e83dd-d694-40ee-80fb-f2f5f8a07742.png)

**Comment by hardillb** [comment]
How are you looking at the logs?

Also there must be an error if Node-RED fails to start with the `//` removed from that line (which REALLY shouldn't cause it not to start)

**Comment by Anthom12** [comment]
I got this error removing the // from line 327 
![image](https://user-images.githubusercontent.com/95219857/143906984-2bc33ddb-5bb7-45a0-bcc7-0ee58498d9cf.png)

**Comment by hardillb** [comment]
Change line 327 to 

    console.log("err: ", err);

**Comment by Anthom12** [comment]
I have no more errors - nodered is launching now .
When I update the list of devices I have this error in the log: 
err: Error: certificate has expired

**Comment by hardillb** [comment]
What version of NodeJS and Node-RED are you running?

**Comment by hardillb** [comment]
Actually it's in the log you sent 8.17.0 which is **VERY VERY** out of support. NodeJS has a built in set of root CA certificates and the CA cert required to verify the connection to my server is expired. 

You will need to upgrade to a supported version of NodeJS (You should probably upgrade Node-RED as well)

You can try running with the following environment variable exported:

    NODE_TLS_REJECT_UNAUTHORIZED=0

Which might work as a temp fix, but this is not something you should run in a "production" environment as it basically turns off all the protection TLS grants.

https://stackoverflow.com/questions/69400610/axios-fails-with-certificate-has-expired-when-certificate-has-not-expired

**Comment by Anthom12** [comment]
Ok thank for your help
